docs/proposals/20210203-unmanaged-infrastructure.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Externally Managed cluster infastructure
+authors:
+  - "@enxebre"
+  - "@joelspeed"
+  - "@alexander-demichev"
+reviewers:
+  - "@vincepri"
+  - "@randomvariable"
+  - "@CecileRobertMichon"
+  - "@yastij"
```

Comment:
Pleas add all the people who provided feedback on this PR

Reply:
I included you, please let me know if you are missing anyone else.